pyre/pyre-interpreter/src/module/_multibytecodec/app_multibytecodec.py

```
@@ -0,0 +1,109 @@
+        return _encode(self.name, input, errors, True)
+
+    def decode(self, input, errors="strict"):
+        return _decode(self.name, input, errors, True)
+
+
+class MultibyteIncrementalDecoder:
+    def __init__(self, errors="strict"):
+        self.errors = "strict" if errors is None else errors
+        self.pending = b""
+
+    def decode(self, object, final=False):
```

Ruff (0.16.1)

[error] Function argument `object` is shadowing a Python builtin

(A002)